<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAFailProfileSawToothWgt</class>
 <widget class="QWidget" name="GTAFailProfileSawToothWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="offsetLE"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="periodLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>phase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phaseLE"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
